2-Regression/1-Tools/translations/README.az.md

```diff
@@ -0,0 +1,227 @@
+
+    Gördüyünüz kimi bu məlumat, 10 elementli massivi olan 442 bənddən ibarətdir:
+
+    ```text
+    (442, 10)
+    [ 0.03807591  0.05068012  0.06169621  0.02187235 -0.0442235  -0.03482076
+    -0.04340085 -0.00259226  0.01990842 -0.01764613]
+    ```
+
+    ✅ Data və reqressiya hədəfi arasındakı əlaqə barəsində biraz düşünün. Xətti reqressiya X xüsusiyyəti ilə y dəyişəni arasındakı əlaqəni təxmin edir. Dokumentasiyada şəkərli diabet data massivindəki [hədəfi](https://scikit-learn.org/stable/datasets/toy_dataset.html#diabetes-dataset) tapa bilərsiniz? Bu data massivi hədəf nəzərdə tutulduqda, nəyi təsvir edir?
+
+2. Növbəti olaraq, qrafiki təsvir üçün bu data massivinin 3-cü sütununu seçin. Bunu `:` operatoru ilə bütün sıraları seçərək və daha sonra index(2) ilə bütün sütunları seçərək edə bilərsiniz. Əlavə olaraq, siz məlumatı qrafiki təsvir üçün tələb olunduğuna görə, `reshape(n_rows, n_columns)` istifadə edərək 2 ölçülü massiv formasına də sala bilərsiniz.
+
+   ```python
+   X = X[:, 2]
+   X = X.reshape((-1,1))
+   ```
+
+   ✅ İstənilən vaxt, datanın quruluşunu yoxlamaq üçün məlumatı ekrana çap edə bilərsiniz.
+
+3. Artıq qrafikləşdirilə bilən datanız olduğuna üçün, maşının bu data massivindəki rəqəmlər arasındakı məntiqi bölgünü təyin etməyə yardımçı ola bilib-bilmədiyini görə bilərsiniz. Bunun üçün hər iki (X) və hədəf (y) datalarını test və öyrətmə massivlərinə ayırmağınıza ehtiyacınız var. Scikit-learn-ün bunu etmək üçün sadə bir yolu var; istənilən nöqtədə test datanızı ayıra bilərsiniz.
+
+   ```python
+   X_train, X_test, y_train, y_test = model_selection.train_test_split(X, y, test_size=0.33)
+   ```
+
+4. Artıq modelinizi öyrətmək üçün hazırsınız! Xətti reqressiya modelinizi yükləyin və onu `model.fit()` istifadə edərək, X və y öyrətmə massivləri ilə öyrədin:
+
+    ```python
+    model = linear_model.LinearRegression()
+    model.fit(X_train, y_train)
+    ```
+
+    ✅ `model.fit()` TensorFlow kimi birçox MÖ kitabxanasında görə biləcəyiniz bir funksiyadır.
+
+5. Daha sonra, test datasından və `predict()` funksiyasından istifadə etməklə proqnoz yaradın. Bu proqnoz, data qrupları arasındakı xəttin çəkilməsi üçün istifadə olunacaq.
+
+    ```python
+    y_pred = model.predict(X_test)
+    ```
+
+6. Artıq datanı qrafiklə göstərməyin vaxtıdır. Matplotlib bu tapşırıq üçün çox uyğun bir alətdir. Bütün X və y test məlumatlarının səpələnmə qrafikini yaradın və modelin data qrupları arasında ən uyğun yerə xətti çəkmək üçün proqnozdan istifadə edin.
+
+    ```python
+    plt.scatter(X_test, y_test,  color='black')
+    plt.plot(X_test, y_pred, color='blue', linewidth=3)
+    plt.xlabel('Scaled BMIs')
+    plt.ylabel('Disease Progression')
+    plt.title('A Graph Plot Showing Diabetes Progression Against BMI')
+    plt.show()
+    ```
+
+   ![şəkərli diabetlə bağlı nöqtələri göstərən səpələnmə qrafiki](./images/scatterplot.png)
```

Comment:
All file paths should be updated 1 directory back. `./images/scatterplot.png` -> `../images/scatterplot.png`

Reply:
Not actually. The images folder is located at the same path with translations. You can check it by viewing the previewed version of this translation.